src/components/marketplace/buy_sell.tsx

```diff
@@ -87,8 +91,8 @@ const LabelContainer = styled.div`
     margin-bottom: 10px;
 `;
 
-const Label = styled.label`
-    color: #000;
+const Label = styled.label<{ color?: string }>`
+    color: ${props => (props.color ? props.color : '#000')};
```

Comment:
Personal preference: `props.color || '#000'` is more idiomatic.

Reply:
Done